### ISSUE TITLE
Homepage needs improvements for mobile view.

### ISSUE THREAD

**Issue by Bazif-Khan** [kabanero-io/kabanero-website#212]
#### Describe the bug
In mobile view, the homepage needs some improvements. e.g. text should be aligned and needs padding as well at some places.

#### Screenshots
![image](https://user-images.githubusercontent.com/51790085/65437343-8813e800-de1b-11e9-8dc1-ca5a10b1a4bb.png)
![image](https://user-images.githubusercontent.com/51790085/65437355-8d713280-de1b-11e9-94ad-0395f58afe10.png)
![image](https://user-images.githubusercontent.com/51790085/65437379-96620400-de1b-11e9-98e2-1a5ae9ded25c.png)
![image](https://user-images.githubusercontent.com/51790085/65437389-99f58b00-de1b-11e9-8103-bda6cb84867f.png)

**Comment by Bazif-Khan** [comment]
@alohr51 can I start the work on this? if others are not already working on this.

**Comment by alohr51** [comment]
@Bazif-Khan yup go ahead, just a warning though. There are currently 2 homepages (index.html and index-new.html) due to our new redesign that is being _slowly_ rolled out, the index-new.html isn't live, but once content gets updated it will be the new one and index.html will be replaced with it.

At this point in time you would have to update both (or you can wait until index-new.html takes over - they are pretty similar) so you don't have to do it twice, or update both- up to you! thanks for helping out.

**Comment by Bazif-Khan** [comment]
@alohr51 thanks for letting me know. I'll wait until index-new.html takes over. Could you please let me know when index-new.html is live?

**Comment by marikaj123** [comment]
@alohr51 - What are the next steps for the issue to get close,d as it is assigned for Oct-7.

**Comment by marikaj123** [comment]
@SueChaplain - Do you have what you need to complete the ID review.

**Comment by SueChaplain** [comment]
@marikaj123 - see https://github.com/kabanero-io/kabanero-website/issues/212#issuecomment-534148564 above. The work to make the change will be wasted if there is a new home page in development but I have no idea what the timescale is for this. @alohr51 - do you have an estimate please? If the review of the new home page is backed up behind a lot of other work, @Bazif-Khan could go ahead with this one now, just to make the site look as good as possible in the interim.

**Comment by SueChaplain** [comment]
@marikaj123 
The status of this work is not green. It is not started. 

I do not want our limited resources working on something that might be wasted effort.
If there is no answer to the question I raised about timescales for the new homepage that is in progress, I would like to see this issue removed from the release 0.2 milestone.

@alohr51 FYI

This is also not an issue that is appropriate for the ID-review tag. It is web development effort to improve the mobile layout of the page. Not something the ID team would review, but rather something the designer would review.

**Comment by alohr51** [comment]
I would take it off the 0.2 release tag and track it for next release, its not ready until we get content from management which has proven very difficult

**Comment by marikaj123** [comment]
yes, if it will not go into release v0.2 the please remove the tag.

**Comment by marikaj123** [comment]
@alohr51 and @stephenkinder and @SueChaplain - Is the work targted for release 0.3.0?

**Comment by alohr51** [comment]
@marikaj123 no, the website is running on its own release schedule

**Comment by marikaj123** [comment]
OK, removing the release tag then.
